docs/getting-started/user-guides/dcrwallet-tickets.md

```diff
@@ -100,9 +100,9 @@ Your `ticketfee` is the DCR/kB rate you'll pay to have your ticket purchase be i
 1. During startup by adding `ticketfee=<fee rate>` to the `[Application Options]` of your `dcrwallet.conf`.
 2. While your wallet is running, using the `dcrctl --wallet setticketfee <fee rate>` command. This is not a permanent setting and will default to 0.01 every time your wallet is restarted unless a ticketfee is specified in `dcrwallet.conf`.
 
-Why are ticketfees important? Usually the default fee of 0.01 is enough to get your tickets mined. The exception is during a fee war event. When the ticket price falls very low, demand outpaces supply (there are only a maximum of 2880 tickets available at each price interval). This creates a fee war, with stakeholders competing to get their tickets mined before the price changes.
+Why are ticket fees important? Usually the default fee of 0.01 is enough to get your tickets mined. The exception is during a "fee war" which can occur when ticket demand outpaces supply (there are only a maximum of 2880 tickets available at each price interval). This creates a situation where stakeholders increase their ticket fees in order to get their ticket purchases mined ahead of others who are offering lower fees.
```

Comment:
default ticket fee is 0.001

Reply:
I would also add a note here that mentions that 'fee wars' aren't really an issue anymore due to the new sdiff algorithm

Reply:
But in general I think we can remove all talk of fee wars in general.  It is just increased demand which could lead to increased ticket fees